lib/y2_logs_helper.pm

```diff
@@ -101,12 +101,20 @@ sub accept_license {
 }
 
 sub verify_license_translations {
-    return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
+    #return if (is_sle && get_var("BETA"));
```

Comment:
unwanted comment here and PR is not in WIP (just a friendly reminder)

Reply:
now commit in WIP but not in the title in github (or with github label) very unlikely to spot for the reviewer.

Reply:
e308b008a
it is not WIP. to run the VR i have to comment out this line. Which i forget to uncomment everytime. Anyway no VR for the latest commit. changes where trivial so i dont think we need another one